api/util.py

```diff
@@ -80,6 +83,17 @@ def commit_file(dbc, _id, datainfo, filepath, data_path, force=False):
     """Insert a file as an attachment or as a file."""
     filename = os.path.basename(filepath)
     fileinfo = datainfo['fileinfo']
+    log_path = os.path.join(os.path.split(data_path)[0], 'logs')
+    # XXX Eventually it would be good to find a more principled/constant
+    # way to do this
+    if os.path.isdir(log_path):
+        hdlr = logging.FileHandler(os.path.join(log_path, 'commit_file.log'))
+        fmt = logging.Formatter(fmt=log_fmt, datefmt=datefmt)
+        hdlr.setFormatter(fmt)
+        hdlr.setLevel(logging.DEBUG)
+        log.addHandler(hdlr)
```

Comment:
I agree it's a hack and I don't like it either, but none of these logging
messages got saved currently as far as I could find. The first thing I
checked was uwsgi and didn't see anything, and I think I actually grep
-R'ed to check and didn't find the messages anywhere. I think it's better
to have the hack in place, working but ugly, than not have the output at
all, but I'll get rid of it if you want. Ideally we could do it in a more
principled way, but I don't get how the pieces for together well enough yet
to see what that is...